File: Plugins/Aspose_Tasks_Java_for_Ruby/lib/asposetasksjava/Projects/projectversion.rb
module Asposetasksjava
  module ProjectVersion
    def initialize()
        data_dir = File.dirname(File.dirname(File.dirname(File.dirname(__FILE__)))) + '/data/'
                
        # Instantiate project object
        project = Rjb::import('com.aspose.tasks.Project').new(data_dir + 'Sample.xml')

        # Display project version property
        puts "Project Version : " + project.get(Rjb::import('com.aspose.tasks.Prj').SAVE_VERSION).to_string
        puts "Last Saved : " + project.get(Rjb::import('com.aspose.tasks.Prj').LAST_SAVED).to_string
    end
  end
end

File: Plugins/Aspose_Tasks_Java_for_PHP/src/aspose/tasks/WorkingWithProjects/ProjectVersion.php
<?php
namespace Aspose\Tasks\WorkingWithProjects;

use com\aspose\tasks\Project as Project;
use com\aspose\tasks\Prj as Prj;

class ProjectVersion{

    public static function run($dataDir=null){

        # Instantiate project object
        $project = new Project($dataDir . 'Sample.xml');

        # Display project version property
        $prj=new Prj();
        print "Project Version : " . (string)$project->get($prj->SAVE_VERSION).PHP_EOL; //->to_string
        print "Last Saved : " . (string)$project->get($prj->LAST_SAVED).PHP_EOL;
    }

}
?>